selfdrive/locationd/test/test_paramsd.py

```diff
@@ -6,7 +6,6 @@
 import cereal.messaging as messaging
 from cereal import car
 from common.params import Params
-from selfdrive.locationd.paramsd import load_params
```

Comment:
import here will cause `self.localizer = self.lib.localizer_init()` failed in `selfdrive/locationd/test/test_locationd.py` with the following assertion:

> python3.8: rednose/helpers/ekf_sym.cc:14: EKFS::EKFSym::EKFSym(std::string, Map<EKFS::MatrixXdr>, Map<Eigen::VectorXd>, Map<EKFS::MatrixXdr>, int, int, int, int, int, std::vector<int>, std::vector<int>, std::vector<std::string>, double): Assertion `this->ekf' failed.